app/config/urls.py

```diff
@@ -15,13 +15,27 @@
 """
 from django.conf import settings
 from django.contrib import admin
+from django.http import JsonResponse
 from django.urls import include
 from django.urls import path
 
+from rest_framework.authtoken.views import obtain_auth_token
+
+STAC_BASE = settings.STAC_BASE
+
+
+def checker(request):
+    data = {"success": True, "message": "OK"}
+
+    return JsonResponse(data)
+
+
 urlpatterns = [
-    path('', include('stac_api.urls')),
-    path('api/stac/admin/', admin.site.urls),
     path('', include('django_prometheus.urls')),
+    path('checker', checker, name='checker'),
+    path('get-token', obtain_auth_token, name='get-token'),
```

Comment:
Oops still not the correct path from past discussion:
- original: `/api/stac/v0.9/get-tocken`
- new proposal from @boecklic : `/api/stac/get-tocken`
I would be ok to move it at the root, but I'm not sure if this is a good idea, @boecklic and @hansmannj  what are your opinion ?

Reply:
OK for me, it can well be decoupled from the API version, so no need for the `/v0.9/`-part for me.
This would imply, that a token remains also valid for future API versions, which is OK for me.

Reply:
`/api/stac/v0.9/get-token` and `/api/stac/get-token` both work, `/get-token` doesn't work (since not visible to the outside). 

If it makes code simpler (i.e. no exception regarding path parsing to determine cache header) I'd prefer `/api/stac/get-token` (plus it makes even more sense, get-token is independant of the stac version )

Reply:
@ltshb sorry